From 039097dd8059de8a0f5b2a6a7cf4f3e5f7c8c4f0 Mon Sep 17 00:00:00 2001
From: Aswin M <Aswin_M@comcast.com>
Date: Tue, 24 May 2022 10:59:26 +0000
Subject: [PATCH] HX44X-149 : Bring in Voice Support

Reason for change: Added overlay support for speaking state.
Test Procedure: Enable necessary configurations and build.
Risks: Low
License: Inherited
Upstream-Status: Pending
Source: RDKM

Signed-off-by: Aswin M <Aswin_M@comcast.com>
---
 src/VoiceToApps.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/VoiceToApps.cpp b/src/VoiceToApps.cpp
index 07e07d7..153fe23 100644
--- a/src/VoiceToApps.cpp
+++ b/src/VoiceToApps.cpp
@@ -216,6 +216,9 @@ int voiceToApps::controlSmartScreenOverlay(VoiceSDKState state,  bool audioState
        else if(state == VoiceSDKState::VTA_IDLE && ! audioStatePlaying) {
 	  oState = OverlayState::HIDE;
        }
+       else if(state == VoiceSDKState::VTA_SPEAKING) {
+          oState = OverlayState::SHOW;
+       }
        else {
           oState = OverlayState::DIMMED;
        }
-- 
2.17.1

